pyproject.toml

```diff
@@ -1,2 +1,108 @@
 [build-system]
 requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
```

Comment:
can drop `wheel` because we're using `build` now instead.

Also, it'd be worth moving to these versions now: `["setuptools>=61.0", "setuptools_scm[toml]>=6.2"]`

Reply:
this will ensure proper build isolation, versioning, and script/entrypoint installs